application/src/js/cms/rd-table.js

```diff
@@ -54,7 +54,7 @@ function appendSimpleTableHeader(table_html, tableObject) {
     if (tableObject['category_caption'] == null) {
         header_html = "<thead><tr><th></th>";
     } else {
-        header_html = "<thead><tr><th>" + tableObject.category_caption + "</th>";
+        header_html = "<thead><tr><th id='index-column-caption'>" + tableObject.category_caption + "</th>";
```

Comment:
Can we use double-quotes for HTML attributes please? (either works, but double-quotes are more conventional).